demos/html5/generic/test/e2e-insert-formula.test.js

```diff
@@ -0,0 +1,68 @@
+const puppeteer = require('puppeteer');
+
+const timeout = 10000;
+
+// Declare browser and page that will contain the demo page
+let browser;
+let page;
+
+/**
+ * This section is dedicated to tests the insertion of a formula in the froala3 demo.
+ * It should test the MT formula insertion and the CT formula insertion
+ */
+describe('Insert Formula. TAG = Insert',
+  () => {
+    // Execute before all the file tests to define the browser with puppeteer
+    beforeAll(async () => {
+      browser = await puppeteer.launch({
+        headless: true,
+        args: [
+          '--disable-gpu',
+          '--disable-dev-shm-usage',
+          '--no-sandbox',
+          '--disable-setuid-sandbox',
+        ],
+        timeout: 10000,
+        devtools: false,
+        slowMo: 0,
+        defaultViewport: null,
+      });
+    });
+
+    // Execute afterall the tests of the file to close the browser
+    afterAll(async () => {
+      browser.close();
+    });
+
+    // Execute before each test of the file to open the demo page
+    beforeEach(async () => {
+      jest.setTimeout(10000);
```

Comment:
I've commented this line, along with line 25, and the test passed anyway.
Both on the demo's folder and the genera 'npm test' command.
So I guess the line 39, could be removed.

Reply:
I forgot to add that it seems to don't impact on the test execution time, either. :shrug:

Reply:
This function sets the maximum time before the test times out, in case something doesn't work (so it's NOT like `window.setTimeout`). I don't know if @carlawiris set this number for any specific reason but I can guess that maybe the tests may be slower on Travis. Anyway, I don't see the harm in having a high timeout.

Reply:
Thanks, @xaviripo 
This is super good to know.

Reply:
https://jestjs.io/docs/en/jest-object#jestsettimeouttimeout